Cargo.toml

```diff
@@ -7,7 +7,7 @@ name = 'pack_encrypted'
 harness = false
 
 [[example]]
-name = 'basic'
+name = 'advanced_params'
```

Comment:
I would expect all examples must be in Cargo.toml

Reply:
fixed